deepchecks/nlp/utils/text.py

```diff
@@ -117,6 +117,15 @@ def normalize_text(
     return text_sample
 
 
+def cut_string(input_str: str, cut_length: int = 200) -> str:
+    """Cut a string to 200 characters, but cut only at whitespaces."""
+    if len(input_str) > cut_length:
```

Comment:
Why does it matter if it's in whitespace or not?
and out of curiousity - why 200?

Reply:
whitspace - so not to cut mid-word
200 - just so?

Reply:
yeah, but what does it matter if you cut mid-word? you're just doing this to compare duplicates...
And was curious, seems like an odd number